src/main/java/org/zalando/nakadi/service/subscription/state/StartingState.java

```diff
@@ -106,7 +102,7 @@ private void initializeStream() {
         try {
             getContext().registerSession();
         } catch (Exception ex) {
-            TracingService.logErrorInSpan(startingStateScope, ex.getMessage());
+            TracingService.logErrorInSpan(getContext().getCurrentSpan(), ex.getMessage());
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs